test/whiteout.bats

```diff
@@ -69,19 +69,28 @@ fulldir:
     rm -rf /a1
     mkdir /a1
     touch /a1/newfile
+
+redo:
+  from:
+    type: built
+    tag: bb
+  run: |
+    rm -rf /a1
+    mkdir /a1
+    touch /a1/newfile
+
 EOF
 
   stacker build
 
   # "bb" should not have a whiteout entry for /a1
   md=$(cat oci/index.json | jq .manifests[0].digest | sed s/sha256://g | tr -d \")
   ld=$(cat oci/blobs/sha256/"$md" | jq .layers[-1].digest | sed s/sha256://g | tr -d \")
-  "bsdtar -tvf oci/blobs/sha256/$ld | grep '.wh.a1'"
```

Comment:
Hi,

these look good.  Though for this particular hunk, would it be better to not remove this bsdtar|grep, but rather turn it into a bsdtar | grep -v '.wh.a1' ?